tests/reexecute/c/vm_reexecute_test.go

```diff
@@ -79,7 +79,9 @@ var (
 	configKey         = "config"
 	defaultConfigKey  = "default"
 	predefinedConfigs = map[string]string{
-		defaultConfigKey: `{}`,
+		defaultConfigKey: `{
+			"allow-missing-tries": true
+		}`,
```

Comment:
This explicitly allows archival execution on top of a non-archival database to support creating height sharded archival DBs.